doc/modules/mod_jingle_sip.md

```diff
@@ -42,3 +104,168 @@ MongooseIM 2.2.x packages are built with OTP 19.3, so they include Jingle/SIP su
+    |                                  |<--------------------|                             |
+    |                                  |     --------------\ |                             |
+    |                                  |     | from: UserA |-|                             |
+    |                                  |     | to: UserB   | |                             |
+    |                                  |     | sid: 123    | |                             |
+    |        session-accept from UserB |     |-------------| |                             |
+    |<---------------------------------|                     |                             |
+    |                                  |                     |                             |
+```
+
+<!---
+object UserA MongooseIM SIPProxy UserB
+UserB->MongooseIM: session-accpet to UserA
+note left of UserB: Jingle stanza \n action:session-accept\nsid: 456
+MongooseIM->SIPProxy: 200 OK
+note right of MongooseIM:from: UserA\nto: UserB\nsid: 456
+SIPProxy->SIPProxy: find corresponding call
+SIPProxy->MongooseIM: 200 OK
+note left of SIPProxy: from: UserA\nto: UserB\nsid: 123
+MongooseIM->UserA: session-accept from UserB
+-->
+
+##### 1.3 Terminating a call
+
+Any Jingle session (accepted or not) can be terminated by sending a Jingle stanza with action `session-terminate` and a reason.
+In the SIP world it's more complex.
+See the following examples for more information.
+
+###### 1.3.1 Terminating an accepted call
+
+The easiest scenario is when the call was accepted as in [1.2](#12-signaling-session-accept-to-other-xmpp-user-via-sip-proxy).
+In this case one of the users sends a `session-terminate` Jingle action with reason `success`.
+This is translated to a SIP `BYE` request with `to` and `from` headers set appropriately -
+`from` is the user who wants to terminate the call and `to` is the user on the other end of the session.
+The `BYE` request is sent to the SIP Proxy and then to the other user in a similar way to session acceptance.
+
+###### 1.3.2 Terminating an unanswered call by initiator
+
+To terminate the call before it's accepted, the initiator sends a Jingle `session-terminate` stanza with a reason `decline`.
+Then MongooseIM translates this to a SIP `CANCEL` request which is sent to the SIP Proxy.
+
+###### 1.3.3 Rejecting the call
+
+When the invitee wants to terminate the call, on the XMPP level this is also a Jingle `session-terminate` stanza with a reason `decline`.
+MongooseIM translates this to SIP `486 Busy Here` **Response** (because this is a response to the invite request).
+
+#### 2. Establishing a session with a SIP user
+
+Establishing a session with a SIP user (or a SIP entity) works the same as in the previous section.
+The only difference is that the SIP Proxy will not call MongooseIM back (as it may happen for call to other XMPP user).
+Instead the SIP message send by MongooseIM to SIP Proxy will be delivered directly to the SIP user's device.
```

Comment:
Instead the SIP message sent by MongooseIM to SIP Proxy will be delivered directly to the SIP user's device.